### ISSUE TITLE
Failed to load the "Paynl_Payment/payment/default" template

### ISSUE THREAD

**Issue by abvdveen** [paynl/magento2-plugin#201]
After enabling the Paynl module, testing it gives the error 'Failed to load the "Paynl_Payment/payment/default" template requested by "checkout.steps.billing-step.payment.payments-list.paynl_payment_ideal"' in the browser console in the payment step at checkout.
Installed it on my staging site yesterday, without any problems.

**Comment by woutse** [comment]
Thanks for your message. Unfortunately we have never seen or heard about this issue before. Please try to reinstall the module.

**Comment by abvdveen** [comment]
@woutse Thanks for the feedback. Have removed the software and re-installed it via composer. After that initially still had issues: after enabling iDeal in the admin and saving the config, it kept reporting iDeal was disabled. Then I decided to enable 
Overboeking and got a similar error at checkout as I reported, now requested by "checkout.steps.billing-step.payment.payments-list.paynl_payment_overboeking".

Then again enabled iDeal, so both iDeal and Overboeking were enabled, and now the error at checkout disappeared and both iDeal and Overboeking were available.

Finally disabled Overboeking and now everything works as expected. But something, somewhere is wobbly :)
